README.md

```diff
@@ -193,6 +193,14 @@ Example:
 $ python sumo_mgmt.py -url https://api.sumologic.com/api/v1/ -accessid [YOUR ACCESS ID] -accesskey [YOUR ACCESS KEY] -deleteOfflineCollectors [aliveBeforeDays]
 ```
 
+### Assign Ingest Budget to Collectors
+
+A list of collectors based on filter can be assigned Ingest Budget
```

Comment:
can we change this wording to:

A filtered set of collectors can be assigned to an [Ingest Budget](https://help.sumologic.com/Manage/Ingestion-and-Volume/Ingest_Budgets) by using the `-assignBudget` command with the ID of the budget you'd like to assign.

Reply:
Done